Engine/EditorUI/Panel/PanelPopups.cpp

```diff
@@ -36,6 +36,7 @@ void PanelPopups::RenderAssetsLoadingPopup()
 	{
 		ImGui::Text("Loading Assets");
 		float progress = (float)App->resources->thread_comunication.loaded_items / App->resources->thread_comunication.total_items;
+		assert(progress <= 1);
```

Comment:
what you want to ensure with this assert?

Reply:
Just to debug if number of imported/remaining files was computed correctly. Spoiler: it's not 😄